modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/representative_form_upload_controller.rb

```diff
@@ -19,20 +19,68 @@ def submit
 
       def upload_scanned_form
         authorize(nil, policy_class: RepresentativeFormUploadPolicy)
-        attachment = PersistentAttachments::VAForm.new
-        attachment.form_id = params['form_id']
-        attachment.file = params['file']
-        raise Common::Exceptions::ValidationErrors, attachment unless attachment.valid?
+        handle_file_upload(PersistentAttachments::VAForm)
+      end
+
+      def upload_supporting_documents
+        authorize(nil, policy_class: RepresentativeFormUploadPolicy)
+        handle_file_upload(PersistentAttachments::VAFormDocumentation)
+      end
+
+      private
+
+      def handle_file_upload(form_type)
+        attachment = create_form(form_type)
```

Comment:
Can we update our variable and method names to use the noun "attachment" when that is the object we're manipulating? We can avoid the ambiguity of the nouns "form" and "file" when they are inaccurate.

Reply:
To be more explicit about the concepts we're manipulating here: we're dealing with attachments. Attachments can either be forms or documentations. One of the properties of attachments are their files.